redskyctl/internal/commands/commands.go

```diff
@@ -84,6 +85,7 @@ func NewRedskyctlCommand() *cobra.Command {
 	rootCmd.AddCommand(kustomize.NewCommand())
 	rootCmd.AddCommand(version.NewCommand(&version.Options{Config: cfg}))
 	rootCmd.AddCommand(docs.NewCommand(&docs.Options{}))
+	rootCmd.AddCommand(export.NewCommand(&export.Options{Config: cfg}))
```

Comment:
Since the command ordering comes from this declaration now, this should be up under the generate command.